assets/queries/openAPI/response_object_incorrect_ref/query.rego

```diff
@@ -5,13 +5,15 @@ import data.generic.openapi as openapi_lib
 CxPolicy[result] {
 	doc := input.document[i]
 	openapi_lib.check_openapi(doc) != "undefined"
-	response := doc.paths[n][oper].responses[code]
-	openapi_lib.content_allowed(oper, code)
-	openapi_lib.incorrect_ref(response["$ref"], "responses")
+
+	[path, value] := walk(doc)
+
+	ref := value.responses[code]["$ref"]
```

Comment:
I think we should ignore `components.responses`, since it can have an external reference...